src/views/Dashboard.vue

```diff
@@ -336,12 +334,6 @@ section header {
   }
 }
 
-.pending-chip {
-  max-width: 3.75rem;
-  height: 1rem;
-  overflow: visible;
-  font-size: 8px;
-}
```

Comment:
It turns out all of these (except overflow) were overridden by other styles. And overflow did nothing in this case.